src/integration-test/tests/citizen/defence/pages/defendant-claim-details.ts

```diff
@@ -13,9 +13,14 @@ export class DefendantClaimDetails {
     I.click('View claim')
   }
 
-  checkClaimData (claimReference: string): void {
+  checkClaimData (claimReference: string, getTotal: boolean = true, getTotalWithInterest: boolean = false): void {
     I.see(claimReference)
-    I.see(claimAmount.getTotal())
+    if (getTotal) {
```

Comment:
That solution is not very flexible. Passing bunch of true / false flags impacts readability as well. If this is check function we should pass claim object to it and expect that UI shows expected values.